blog/2025-03-31-envio-dev-update-march-2025.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Envio Developer Update March 2025
+sidebar_label: Envio Developer Update March 2025
+slug: /envio-developer-update-march-2025
+---
+
+<img src="/blog-assets/envio-dev-update-march-2025.png" alt="Cover Image Envio Developer Community Update March 2025" width="100%"/>
+
+<!--truncate-->
+
+March was madness! We’re excited to roll out HyperIndex v2.14.0 with RPC failover support, LogTui for fast chain scans, and integrations with V12, XDC Network, Monad, and Chiliz. With new tools, powerful partnerships, tutorials, and more, we’re empowering developers to build smarter and faster than ever. Let’s dive into the latest updates!
+
+
+## 🚀 HyperIndex Version 2.14.0 is now available 🚀
+
+<img src="/blog-assets/envio-dev-update-march-1-2025.png" alt="v2.14.0" width="100%"/>
+
+
+HyperIndex v2.14.1 is live! This update brings enhanced reliability with RPC failover support—ensuring 100% uptime for your indexer. If HyperSync becomes unavailable, your indexer will automatically switch to an RPC provider.
+
+
+#### What's new?
+
+
+* **RPC as a fallback** → If the primary data source doesn’t receive a new block in 20 seconds, your indexer seamlessly switches to an RPC provider.
+* **New RPC configuration syntax** → Easily add redundancy with a single field.
+* **Advanced control** → Define multiple RPC endpoints, set fallback priorities, and customize block intervals.
+
+
+
+#### How to enable it?
+
+Simply add an RPC field to your network configuration:
+
+
+
+* If HyperSync is available for a chain, it remains the primary data source, with RPC as a fallback.
+
+For chains without HyperSync support, RPC becomes the primary data source.
+
+
+```
+networks:
+  - id: 137 # Polygon
+    start_block: 0
++   rpc: https://eth-mainnet.your-rpc-provider.com
+    contracts:
+      - name: PolygonGreeter
+        address: 0x9D02A17dE4E68545d3a58D3a20BbBE0399E05c9c
+```
+
+
+**Advanced RPC Configuration:**
+
+Want more control? You got it. Now you can explicitly define primary vs. fallback RPC providers:
+
+
+```
+networks:
+  - id: 137 # Polygon
+    start_block: 0
++   rpc: https://eth-mainnet.your-rpc-provider.com
```

Comment:
The example is not correct. It's the same as the above one, but should be different